test/os2lana_cfg.py

```diff
@@ -291,6 +299,7 @@
     *process.evtcleaner
     *process.cleanedEvents
     *process.ana
+#    *process.massReco
     *process.finalEvents
     )
```

Comment:
I do because I want to read the weights that are calculated in evtcleaner and modified in ana